src/components/table/table.stories.mdx

```diff
@@ -0,0 +1,359 @@
+  <Story name="Sortable">
+    {() => {
+      const [state, setState] = useState({
+        sortOrder: "asc",
+        sortedColumn: "name",
+      });
+      const handleChange = (e, tableOptions) => {
+        const { sortOrder, sortedColumn } = tableOptions;
+        setState((prevState) => ({
+          ...prevState,
+          sortOrder,
+          sortedColumn,
+        }));
+      };
+      const getDataToDisplay = () => {
+        let records = countriesList.toJS();
+        if (state.sortOrder === "desc" && state.sortedColumn.length) {
+          records = records.reverse();
+        }
+        return records.slice(0, 9);
+      };
+      return (
+        <Table
+          onChange={handleChange}
+          onSort={() => {}}
+          sortOrder={state.sortOrder}
+          sortedColumn={state.sortedColumn}
+        >
+          <TableRow as="header">
+            <TableHeader sortable name="name">
+              Country
+            </TableHeader>
+            <TableHeader sortable name="code">
+              Code
+            </TableHeader>
+          </TableRow>
+          {getDataToDisplay().map((country) => (
+            <TableRow key={country.value}>
+              <TableCell>{country.name}</TableCell>
+              <TableCell>{country.value}</TableCell>
+            </TableRow>
+          ))}
+        </Table>
+      );
+    }}
+  </Story>
+</Preview>
+
+### Selectable
+
+<Preview>
+  <Story name="Selectable">
+    {() => (
+      <Table
+        selectable
+        onSelect={(selectedRow) => console.log(selectedRow)}
+        actions={{
+          delete: {
+            icon: "bin",
+            onClick: () => {},
+          },
+          settings: {
+            icon: "settings",
+            onClick: () => {},
+          },
+        }}
+        actionToolbarChildren={(context) => (
+          <>
+            <Button
+              disabled={context.disabled}
+              key="single-action"
+              onClick={() => {}}
+            >
+              Test Action
+            </Button>
+            <MultiActionButton
+              text="Actions"
+              disabled={context.disabled}
+              key="multi-actions"
+            >
+              <Button onClick={() => {}}>foo</Button>
+              <Button onClick={() => {}}>bar</Button>
+              <Button onClick={() => {}}>qux</Button>
+            </MultiActionButton>
+          </>
+        )}
+      >
+        <TableRow as="header" uniqueID="header">
+          <TableHeader>Country</TableHeader>
+          <TableHeader>Code</TableHeader>
+        </TableRow>
+        {countriesList
+          .toJS()
+          .slice(0, 3)
+          .map((country) => (
+            <TableRow key={country.value} uniqueID={country.value}>
+              <TableCell>{country.name}</TableCell>
+              <TableCell>{country.value}</TableCell>
+            </TableRow>
+          ))}
+      </Table>
+    )}
+  </Story>
+</Preview>
+
+### Highlightable
+
+<Preview>
```

Comment:
This story should not have a chromatic snapshot because it does not show anything more than **Basic**  does.